app/controllers/reviews_controller.rb

```diff
@@ -23,9 +23,7 @@ def update
     review
     park
```

Comment:
Remove both `review` and `park`, you should use a callback to run this for the appropriate actions